feat: add customer logos

hofmann-felix opened the pull request:
### What's included?
* Customer logo SVGs are added to the project locally
* `<Customers />`component which shows and aligns the logos on the landingpage 

### Preview
* URL: https://satellytescomfeatmaindesignupd-featcustomerlogos66676.gatsbyjs.io/
* Mobile: 
<img width="436" alt="Bildschirm­foto 2023-03-03 um 13 34 08" src="https://user-images.githubusercontent.com/57712895/222721416-887990f6-1fad-4c07-821b-55583bce72af.png">

* Desktop:
<img width="2325" alt="Bildschirm­foto 2023-03-03 um 13 34 38" src="https://user-images.githubusercontent.com/57712895/222721490-88e20205-aa55-425b-b6ad-6852b781d9f9.png">

hofmann-felix committed:
feat: add single customer logo

hofmann-felix committed:
feat: add customer logos

hofmann-felix committed:
feat: adjust margin

ghost commented:
## :white_check_mark: satellytes.com-main deploy preview ready
* [Deploy preview](https://satellytescommain21751-featcustomerlogos.gatsbyjs.io)
* [Build logs](https://gatsbyjs.com/dashboard/f215130d-15e3-4176-9252-e13166fe48de/sites/674be368-adf0-4ac5-8fae-f9a4493adc51/builds/9077c612-4d34-45fc-8f55-271f717bfaf4/details) · **58s** build time
* Performance **85** :orange_circle: · Accessibility **97** :green_heart: · Best Practices **100** :green_heart: · SEO **86** :orange_circle: — [View full report](https://build-9077c612-4d34-45fc-8f55-271f717bfaf4.gtsb.io/reports/lighthouse/index.html)

georgiee commented on src/pages/index.tsx:
```diff
@@ -100,6 +110,21 @@ export const IndexPageQuery = graphql`
       }
     }
 
+    allContentfulCustomerLogos(
```
So did you create a new contentful model? That should be mentioned in the PR
ideally with some notes about how to manage them (or at least a screenshot?)

georgiee commented on src/components/pages/landingpage/customers.tsx:
```diff
@@ -0,0 +1,72 @@
+        {Logos[7]}
+      </LogoLine>
+    </LogosContainer>
+  );
```
I see that there is a contentful model for logos, but on the other side you rely on the first 8 logos.
+ What happens if I delete one?
+ All logos beyond the count of 8 are ignored

I wonder, wouldn't it be more pragmatic not to adjust the CMS and just display a set of 8 logos locally?
Wouldn't it even easier to display a set of 1 or 2 svg files that contains all logos already, especially because everything is so static here?

georgiee reviewed (changes requested):
Thanks for this, looks great 👍

While I really want to see our clients on the website I would like to discuss your approach. Can you check my comments? Thanks.

hofmann-felix committed:
feat: add customer logos locally

hofmann-felix committed:
chore: revert blog grid change

hofmann-felix committed:
Merge branch 'feat-main-design-update' into feat-customer-logos

ghost commented:
## :white_check_mark: satellytes.com-feat-main-design-update deploy preview ready
* [Deploy preview](https://satellytescomfeatmaindesignupd-featcustomerlogos66676.gatsbyjs.io)
* [Build logs](https://gatsbyjs.com/dashboard/f215130d-15e3-4176-9252-e13166fe48de/sites/6edb236d-9b25-45e0-8a0d-5e044af7c366/builds/30fe5680-f7cd-4d1f-ac09-4e97787f79de/details) · **1m** build time
* Performance **78** :orange_circle: · Accessibility **97** :green_heart: · Best Practices **100** :green_heart: · SEO **86** :orange_circle: — [View full report](https://build-30fe5680-f7cd-4d1f-ac09-4e97787f79de.gtsb.io/reports/lighthouse/index.html)

hofmann-felix commented:
@georgiee I agree with you that the CMS doesn't offer any advantages for the logos and that's why I added them locally. I have reverted the changes in Contentful.

georgiee commented:
Thanks for the changes and keeping it simple. 
There is one level of simplicity left and I would like to ask if this is by purpose.

This would suffice here:
```
<img src="clients.svg">
```

Because our client list will not change very often I think we don't need the individual svgs. A single svg would be enough and makes it even more simple. What do you think?

hofmann-felix commented:
@georgiee In the current implementation, the number of rows adjusts to the viewport (s. video). If this behavior is not necessary and a fixed number of rows is sufficient, I agree with your suggestion. This would make the implementation much simpler. 

https://user-images.githubusercontent.com/57712895/223400564-615e1471-41f3-47b9-9ce2-c2a48b34a61e.mov

georgiee commented:
Thanks for the confirmation. I see the aspects why you chose to have distinct svg files per client. Let's go with this solution. It's always easy to remove code once we realize we don't need that. Thanks.

hofmann-felix commented:
@georgiee I also think there may be a better solution. I'll merge the PR, maybe someone has time for a refactoring later. Thanks for your thoughts and review on this 🙏
